dlt/helpers/airflow_helper.py

```diff
@@ -337,6 +338,7 @@ def log_after_attempt(retry_state: RetryCallState) -> None:
             if self.wipe_local_data:
                 logger.info(f"Removing folder {pipeline.working_dir}")
                 task_pipeline._wipe_working_folder()
+                shutil.rmtree(os.environ["DLT_DATA_DIR"])
```

Comment:
here you fully wipe a folder ie `/tmp`. it is not a private pipeline folder. it is a folder where all the pipelines dirs are stored.
you can erase tons of unrelated data like that. also does not solve the problem in the issue

Reply:
Hm. At least locally it doesn't work like this. Every dag creates (and leaves) a folder in a form of `/tmp/dlt_******`. And when I remove `DLT_DATA_DIR`, it actually points to this folder, not to the entire `/tmp`. So, it only wipes the folder related to this particular run, not touching anything else.

![image](https://github.com/dlt-hub/dlt/assets/26030071/b4aca7a8-c806-4c50-a0a5-46c1f95beec4)

So... I don't know, for now I don't see why this change is incorrect.

Reply:
it is a "minor" problem (user can configure a persistent `DLT_DATA_DIR` for his/her pipelines, this is what this setting is for) - your code will wipe it all.

still the main issue is not addressed, did you read point (1)? is there something wrong with this observation?